packages/@angular/cli/models/schematic-command.ts

```diff
@@ -97,9 +98,9 @@ export abstract class SchematicCommand extends Command {
     const pathOptions = this.setPathOptions(schematicOptions, workingDir);
     schematicOptions = { ...schematicOptions, ...pathOptions };
     const defaultOptions = this.readDefaults(collectionName, schematicName, schematicOptions);
-    // schematicOptions = { ...schematicOptions, ...defaultOptions };
-    schematicOptions.x = defaultOptions;
-    delete schematicOptions.x;
+    schematicOptions = { ...schematicOptions, ...defaultOptions };
+    // schematicOptions.x = defaultOptions;
+    // delete schematicOptions.x;
```

Comment:
Nit: extraneous comments.

Maybe take a note to remove them later to save time atm.

Reply:
Fixed.